### ISSUE TITLE
Cannot find module '@oxc-parser/binding-linux-x64-gnu'

### ISSUE THREAD

**Issue by redexp** [oxc-project/oxc#2369]
Hello, 

1. I'v installed `npm i oxc-parser` (node v20.9, npm v10.1) 
2. created file `test.js` with code `require('oxc-parser').parseSync('var x = 1;');`
3. run it `node test.js`
4. got error 
```
Error: Cannot find module '@oxc-parser/binding-linux-x64-gnu'
...
at Object.<anonymous> (/../node_modules/oxc-parser/index.js:190:31)
```

Code on `node_modules/oxc-parser/index.js:190`
```js
nativeBinding = require('@oxc-parser/binding-linux-x64-gnu')
```
Looking for `node_modules/@oxc-parser/binding-linux-x64-gnu` - present. Native module `parser.linux-x64-gnu.node` - present. I think problem in `package.json` - it not describes where main file of this module (there no `index.js` file)
```json
{
  "name": "@oxc-parser/binding-linux-x64-gnu",
  "version": "0.3.0",
  "author": "Boshen and oxc contributors",
  "license": "MIT",
  "homepage": "https://oxc-project.github.io",
  "bugs": "https://github.com/oxc-project/oxc/issues",
  "repository": {
    "type": "git",
    "url": "https://github.com/oxc-project/oxc.git",
    "directory": "npm/oxc-parser"
  },
  "os": [
    "linux"
  ],
  "cpu": [
    "x64"
  ],
  "libc": [
    "glibc"
  ]
}
```

**Comment by ArnaudBarre** [comment]
Fixed the issue by patching the file and adding `/parser.<platform>.node` to the require call

**Comment by Boshen** [comment]
This should be fixed with the latest oxc-parser@0.4.0 release. 

Sorry for the trouble, I'm not very good at creating npm packages 😅
